### DISCUSSION TITLE
Installation fails at directory creation

### DISCUSSION THREAD

**Discussion by boonware** [fomantic/Fomantic-UI#1789] [General] [answered]
I have followed [this link](https://medium.com/@fomantic/setting-up-fomantic-ui-with-react-f701b68f736c) to install Fomantic UI into a React project. During the install I am prompted 

> Where should we put Semantic UI inside your project?

The default value is `semantic/`. I instead choose a different directory in my project, `src/asset`. This directory exists and the installation is executed inside my project root directory. The installation fails with:

> NPM does not have permissions to create folders at your specified path. Adjust your folders permissions and run "npm install" again

As stated above, the directory already exists and I have confirmed read permissions. Not sure if this is strictly NPM or Fomantic UI related.

**Comment by lubber-de** [comment, marked as answer]
Sounds like a NPM issue. it clearly says 
> "NPM does not have permissions to create folders at". 

> I have confirmed read permissions

The NPM message needs "write" permissons..., not read.
